src/tools/process.cc

```diff
@@ -290,3 +282,25 @@ bool Process::isAlive()
 	return true;
 }
 
+void Process::prepareEnvironmentVariables()
+{
+    Settings* settings = Settings::getInstance();
+    QProcessEnvironment env = QProcessEnvironment::systemEnvironment();
+    if ( !this->filteredEnvironmentVars.empty() ) {
+        for(auto const &i : this->filteredEnvironmentVars) {
+            env.remove(i);
+            LogFileUtils::getInstance()->writeLog(" Filter environment variables: " + (i));
+         }
+    }
+    if (this->executableName == settings->getRsyncName() ) {
+        QString libraryPaths = env.value("LD_LIBRARY_PATH", "");
+        if (env.contains("LD_LIBRARY_PATH") &! libraryPaths.isEmpty() ) {
+            libraryPaths.append( ":" + settings->getApplicationBinDir() );
```

Comment:
please prepend our path, not append, otherwise we may still pick up other DLLs first